release/api/v1alpha1/artifacts.go

```diff
@@ -1,54 +1,62 @@
 package v1alpha1
 
-func (vb *VersionsBundle) Manifests() map[string][]Manifest {
-	return map[string][]Manifest{
+func (vb *VersionsBundle) Manifests() map[string][]*Manifest {
+	return map[string][]*Manifest{
 		"cluster-api-provider-aws": {
-			vb.Aws.Components,
-			vb.Aws.ClusterTemplate,
-			vb.Aws.Metadata,
+			&vb.Aws.Components,
+			&vb.Aws.ClusterTemplate,
+			&vb.Aws.Metadata,
 		},
 		"core-cluster-api": {
-			vb.ClusterAPI.Components,
-			vb.ClusterAPI.Metadata,
+			&vb.ClusterAPI.Components,
+			&vb.ClusterAPI.Metadata,
 		},
 		"capi-kubeadm-bootstrap": {
-			vb.Bootstrap.Components,
-			vb.Bootstrap.Metadata,
+			&vb.Bootstrap.Components,
+			&vb.Bootstrap.Metadata,
 		},
 		"capi-kubeadm-control-plane": {
-			vb.ControlPlane.Components,
-			vb.ControlPlane.Metadata,
+			&vb.ControlPlane.Components,
+			&vb.ControlPlane.Metadata,
+		},
+		"cert-manager": {
+			&vb.CertManager.Manifest,
 		},
 		"cluster-api-provider-docker": {
-			vb.Docker.Components,
-			vb.Docker.ClusterTemplate,
-			vb.Docker.Metadata,
+			&vb.Docker.Components,
+			&vb.Docker.ClusterTemplate,
+			&vb.Docker.Metadata,
 		},
 		"cluster-api-provider-vsphere": {
-			vb.VSphere.Components,
-			vb.VSphere.ClusterTemplate,
-			vb.VSphere.Metadata,
+			&vb.VSphere.Components,
+			&vb.VSphere.ClusterTemplate,
+			&vb.VSphere.Metadata,
 		},
-		"cluster-api-provider-cloudstack": {
```

Comment:
removing this for now, because the bundle-release.yaml doesn't have these manifests yet it seems.

Reply:
it works now with the latest bundle-release.yaml, so adding it back